plugin/pkg/scheduler/factory/factory.go

```diff
@@ -91,12 +95,12 @@ func NewConfigFactory(client *client.Client, schedulerName string) *ConfigFactor
 		PVCLister:        &cache.StoreToPVCFetcher{Store: cache.NewStore(cache.MetaNamespaceKeyFunc)},
 		ServiceLister:    &cache.StoreToServiceLister{Store: cache.NewStore(cache.MetaNamespaceKeyFunc)},
 		ControllerLister: &cache.StoreToReplicationControllerLister{Store: cache.NewStore(cache.MetaNamespaceKeyFunc)},
-		StopEverything:   make(chan struct{}),
+		schedulerCache:   schedulerCache,
+		StopEverything:   stopEverything,
 		SchedulerName:    schedulerName,
 	}
-	modeler := scheduler.NewSimpleModeler(&cache.StoreToPodLister{Store: c.PodQueue}, c.ScheduledPodLister)
-	c.modeler = modeler
-	c.PodLister = modeler.PodLister()
+
+	c.PodLister = &schedulercache.CacheToPodLister{Cache: schedulerCache}
```

Comment:
This is strange construction. Why not just:

  c.PodList = schedulerCache

?